packages/react-scripts/package.json

```diff
@@ -47,9 +47,12 @@
     "find-pkg": "1.0.0",
     "fs-extra": "5.0.0",
     "globby": "7.1.1",
+    "graphql": "0.12.3",
+    "graphql-tag": "2.6.1",
     "html-webpack-plugin": "2.30.1",
     "identity-obj-proxy": "3.0.0",
     "jest": "22.1.2",
+    "jest-transform-graphql": "2.1.0",
```

Comment:
It's kind of frustrating that everybody will "inherit" whole three GQL dependencies after ejecting even if they don't use graphql.

Reply:
I do agree, but not sure what the other option would be.

Reply:
Let's remove this dependency. All it does is literally [one-line call to webpack loader](https://unpkg.com/jest-transform-graphql@2.1.0/index.js), and we could do the same in our Jest transform file.

Reply:
Done ✅